cabot_alert_pagerduty/models.py

```diff
@@ -25,9 +26,19 @@ class PagerdutyAlert(AlertPlugin):
     name = "Pagerduty"
     author = "Mahendra M"
 
+    @property
+    def alert_status_list(self):
+        if not hasattr(self, '_alert_status_list'):
```

Comment:
This is intended to be implemented as a read on demand property to isolate reading from an environment variable that is not normally set and is called when a service is evaluated for whether to alert or not. It may or may not be entirely worth the coding effort, but every little bit helps.